templates/admin/jecoute/data_survey/list_interviewed.html.twig

```diff
@@ -5,11 +5,17 @@
         {{ object.jemarcheDataSurvey.firstName }} {{ object.jemarcheDataSurvey.lastName }}
         <br />
         {{ object.jemarcheDataSurvey.emailAddress }}
-    {% elseif object.isOfCampaignHistory and object.campaignHistory.adherent %}
-        <a href="{{ path('admin_app_adherent_show', { id: object.campaignHistory.adherent.id }) }}">
-            {{ object.campaignHistory.adherent.fullName }}
+    {% elseif object.isOfPhoningCampaignHistory and object.phoningCampaignHistory.adherent %}
+        <a href="{{ path('admin_app_adherent_show', { id: object.phoningCampaignHistory.adherent.id }) }}">
+            {{ object.phoningCampaignHistory.adherent.fullName }}
         </a>
         <br />
-        {{ object.campaignHistory.adherent.emailAddress }}
+        {{ object.phoningCampaignHistory.adherent.emailAddress }}
+    {% elseif object.isOfPapCampaignHistory and object.papCampaignHistory.adherent %}
+        <a href="{{ path('admin_app_adherent_show', { id: object.papCampaignHistory.adherent.id }) }}">
```

Comment:
could you remove the usage of `.adherent` as the relationship does not exist anymore